test/integration/replies_test.rb

```diff
@@ -12,32 +12,16 @@
       context "きづき詳細ページを訪れたとき" do
         before { visit notice_path(@notice) }
 
-        context "正しい値を入力したとき" do
+        context "正しい値を入力して「返信」したとき" do
           before do
             @reply = build(:reply)
             fill_in "reply_body_#{@base_id}", with: @reply.body
+            find(:css, '#new_reply').click_button "返信する"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.